candi/cloud-providers/aws/terraform-modules/master-node/main.tf

```diff
@@ -33,12 +33,18 @@ locals {
 }
 
 resource "aws_ebs_volume" "kubernetes_data" {
-  size            = 150 # To achieve io rate burst limit 450iops, average io rate for etcd is 300iops
-  type            = "gp2"
+  size            = 20 # To achieve io rate burst limit 450iops, average io rate for etcd is 300iops
```

Comment:
I think we should not change disk size. In aws we can not reduce disk size, so we will get destructively changed state and should recreate master nodes.

Reply:
The size is ignored. However, with ignored size and type it is possible to configure anything by hand, which is not the approach we are looking for. It leads to myriads of different variants of etcd disk configurations.